content/en/docs/tasks/administer-cluster/use-cascading-deletion.md

```diff
@@ -302,7 +302,7 @@ For details, read the [documentation for your Kubernetes version](/docs/home/sup
 Run the following command:
 
 ```shell
-kubectl delete deployment nginx-deployment --cascade=false
+kubectl delete deployment nginx-deployment --cascade=orphan
```

Comment:
Hi @Patil2099 .
Just checking if this change is correct for Kubernetes versions *prior* to 1.20.x?
When did the kubectl flag change?

Reply:
Prior to 1.20 this was a boolean flag.

https://github.com/kubernetes/kubernetes/pull/93384/files#diff-3b24d71dec02e50031772f99bffc240106e8098d0a261efb10cd838b6a14a467L122

Reply:
OK. Thanks @eddiezane .
The page preview:
https://deploy-preview-29394--kubernetes-io-main-staging.netlify.app/docs/tasks/administer-cluster/use-cascading-deletion/#set-orphan-deletion-policy

I guess the change in this page should be reverted?
@Patil2099 , Does this make sense?